pkg/prometheus/promcfg.go

```diff
@@ -177,6 +177,11 @@ func (cg *configGenerator) generateConfig(
 		scrapeInterval = p.Spec.ScrapeInterval
 	}
 
+	scrapeTimeout := "10s"
```

Comment:
Whats the reason for the 10seconds here, might have missed something?

Reply:
It's the default timeout. Even changing to 30s the same test fails. I don't think it is related to the change.